hyperspy/io_plugins/hdf5.py

```diff
@@ -425,9 +425,9 @@ def parse_structure(key, group, value, _type, compression):
             try:
                 group.attrs[key] = value
             except:
-                print("The hdf5 writer could not write the following "
-                      "information in the file")
-                print('%s : %s' % (key, value))
+                warnings.warn(
```

Comment:
I'm not familiar with when this would occur: Is this something that is expected to happen often? Or is this only if the user has stored something wrong in metadata? What is the motivation for suppressing the exception?